interface/wx/intl.h

```diff
@@ -33,9 +33,17 @@ struct wxLanguageInfo
     /// language info structure.
     int Language;
 
+    /// Tag for locale in BCP 47-like notation.
```

Comment:
It should be documented that this, and the other new, field(s) is/are only available since wx 3.1.6.

Reply:
Ok, I will adjust the comments.